REFERENCE.md

```diff
@@ -71,146 +59,151 @@ you know the security tradeoffs of ``FIPS-140-2`` compliance.
 ``FIPS`` mode disables the use of ``MD5`` and may require weaker ciphers or
 key lengths than your security policies allow.
 
+@see http://simp.readthedocs.io/en/master/security_mapping/components/simplib/cryptographic_protection/control.html SIMP - Security Control Mapping Cryptographic Protection
+
 Default value: `false`
 
 ##### `firewall`
 
-Data type: `Boolean`
+Data type: `Variant[Boolean,Enum['firewalld']]`
 
-Whether to include SIMP's firewall class ``::iptables``
-and add rules pertinent to each application.
+Include SIMP's firewall class ``iptables`` and add rules pertinent to each
+application
+
+* Set to ``firewalld`` if you want to automatically pass through to the SIMP
+  configuraton of ``firewalld``.
 
 Default value: `false`
 
 ##### `haveged`
 
 Data type: `Boolean`
 
-Whether to include the ``::haveged`` class to ensure adequate
-entropy for key generation
+Include the ``haveged`` class to ensure adequate entropy for key
+generation
+
+@see http://simp.readthedocs.io/en/master/getting_started_guide/ISO_Build/Environment_Preparation.html?highlight=haveged SIMP - Getting Started Environment Preparation
```

Comment:
https://simp.readthedocs.io/en/master/getting_started_guide/Installation_Options/ISO/ISO_Build/Environment_Preparation.html

Reply:
stable in lieu of master